packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -125,7 +126,7 @@ export const MafsGraph = (props: MafsGraphProps) => {
                                 range={state.range}
                             />
                         )}
-                        /* Protractor */
+                        {/* Protractor */}
```

Comment:
Just noticed this was missing the brackets, so added them in since this seems to be a comment.